apps/web/src/views/Info/components/CurrencyLogo/index.tsx

```diff
@@ -16,6 +16,17 @@ const StyledLogo = styled(TokenLogo)<{ size: string }>`
   color: ${({ theme }) => theme.colors.text};
 `
 
+export const CHAIN_NAME_TO_PATH = {
+  ARB: 'arbitrum',
+  POLYGON_ZKEVM: 'polygon-zkevm',
+}
+
+const chainNameToPath = (chainName: MultiChainName) => {
+  if (chainName === 'BSC') return ''
+  if (CHAIN_NAME_TO_PATH[chainName]) return `${CHAIN_NAME_TO_PATH[chainName]}/`
+  return `${chainName.toLowerCase()}/`
+}
+
```

Comment:
duplicated with these one
https://github.com/pancakeswap/pancake-frontend/blob/3d45a094ad6707b7902e1e4e2e9c67b67322e63c/packages/widgets-internal/components/CurrencyLogo/utils.ts#L43-L64

Reply:
ok, let me use this config

Reply:
Done